src/gwt/src/org/rstudio/studio/client/workbench/views/environment/dataimport/DataImportScript.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * DataImportScript.java
+ *
+ * Copyright (C) 2009-16 by RStudio, Inc.
+ *
+ * Unless you have received this program directly from RStudio pursuant
+ * to the terms of a commercial license agreement with RStudio, then
+ * this program is licensed to you under the terms of version 3 of the
+ * GNU Affero General Public License. This program is distributed WITHOUT
+ * ANY EXPRESS OR IMPLIED WARRANTY, INCLUDING THOSE OF NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE. Please refer to the
+ * AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
+ *
+ */
+
+package org.rstudio.studio.client.workbench.views.environment.dataimport;
+
+import org.rstudio.core.client.StringUtil;
+
+public class DataImportScript
+{
+   public DataImportScript()
+   {
+      
+   }
+   
+   public String getImportScript(DataImportOptions options)
```

Comment:
At this point I'm ramping up with the server and session code to extract the preview data; therefore, I would prefer to come back and improve this once I have a better understanding of the constructs I can use from within the R session. So yes, will address this in a subsequent PR and keep this simple code generator for now.